modules/solace/service.go

```diff
@@ -0,0 +1,47 @@
+package solace
+
+type Service struct {
```

Comment:
suggestion: let's add comments to the types 🙏

Reply:
done